sale_loyalty_order_line_link/models/__init__.py

```diff
@@ -1 +1 @@
-from . import sale_order
+from . import loyalty_program, sale_order
```

Comment:
Please use usual import form:

```suggestion
from . import loyalty_program
from . import sale_order
```